src/main/java/com/atherys/towns/AtherysTowns.java

```diff
@@ -255,6 +251,10 @@ public PollFacade getPollFacade() {
         return components.pollFacade;
     }
 
+    public ProtectionFacade getBlockFacade() {
```

Comment:
Should rename this.

Reply:
Figures I missed one, I renamed that thing like 3 times because I kept doing different things with it lol. Thank you.

Reply:
This has been corrected.